src/logger.js

```diff
@@ -59,13 +58,12 @@ function formatError(error, isVerbose) {
 logger.log = function (logLevel, message) {
     if (this.levels[logLevel] >= this.levels[this.logLevel]) {
         var isVerbose = this.logLevel === 'verbose';
-        var prefix = this.prefixes[logLevel] ? this.prefixes[logLevel] + ': ' : '';
 
         if(message instanceof Error) {
             message = formatError(message, isVerbose);
         }
 
-        message = prefix + message + EOL;
+        message = message + EOL;
 
         if (!this.cursor) {
             this.output.write(message);
```

Comment:
I just realized that we should write error messages to stderr instead of output - is that correct?